test/no-new-buffer.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+const RuleTester = require('eslint').RuleTester
+const rule = require('../lib/rules/no-new-buffer')
+
+const ruleTester = new RuleTester();
+
+const allocError = {
+	message: '`new Buffer()` is deprecated, use `Buffer.alloc()` instead.'
```

Comment:
@divyabiyani in this test file we probably want a `2-space` indent, see example: https://github.com/divyabiyani/eslint-plugin-fxa/blob/master/test/async-crypto-random.js